src/manager.cpp

```diff
@@ -425,6 +417,28 @@ void Manager::collectSingleFruVpd(
                 std::string(i_dbusObjPath));
         }
 
+        // Check if host is up and running
+        if (dbusUtility::isHostRunning())
+        {
+            if (!jsonUtility::isFruReplaceableAtRuntime(l_sysCfgJsonObj,
+                                                        l_fruPath))
+            {
+                throw std::runtime_error(
+                    "Given FRU is not replaceable at host runtime. Single FRU VPD collection failed for " +
+                    std::string(i_dbusObjPath));
+            }
+        }
+        else if (dbusUtility::isBMCReady())
+        {
+            if (!jsonUtility::isFruReplaceableAtStandby(l_sysCfgJsonObj,
```

Comment:
Both the FRUs which are replaceable at runtime and standby should be collected here.
FRUs replaceable at runtime are not just replaceable at runtime, they can also be replaced at runtime and any system level below that.

Reply:
ok..got that

Reply:
updated